data/battle_scripts_1.s

```diff
@@ -379,6 +379,7 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectDecorate                @ EFFECT_DECORATE
 	.4byte BattleScript_EffectHit                     @ EFFECT_SNIPE_SHOT
 	.4byte BattleScript_EffectTripleHit               @ EFFECT_TRIPLE_HIT
+	.4byte BattleScript_EffectRecoilHP25			  @ EFFECT_RECOIL_HP_25
```

Comment:
Spaces and alignment inbetween the declaration of the battle script in the table and the comment.

Reply:
Done.